ipalib/rpc.py

```diff
@@ -308,7 +308,8 @@ def json_encode_binary(val, version):
             encoded = encoded.decode('ascii')
         return {'__base64__': encoded}
     elif isinstance(val, Decimal):
-        return {'__base64__': base64.b64encode(str(val))}
+        return {'__base64__': base64.b64encode(
+            str(val).encode('ascii')).decode('ascii')}
```

Comment:
Could we please get rid of this non-sense and return `unicode(val)`? (It is a backward compatible change.)